### PR TITLE
feat: vue-cli support

### PR DESCRIPTION
**Pull request by JessicaSachs**

### CODE REVIEW

File: preprocessor/webpack.js [hunk 1]
```diff
@@ -1,32 +1,15 @@
...
-  }
-}
+const fw = require('find-webpack')
+const webpackOptions = fw.getWebpackOptions()
+
+// TODO: Figure out how to handle dynamic imports
+// Vue CLI's optimization and bundle splitting breaks Cypress
+// but it's necessary for <router-link> lazy loading...
+// and likely other dynamic imports
+delete webpackOptions.optimization
```
Comment (bahmutov) [inline preprocessor/webpack.js:12]: you can probably use `cleanForCypress` method from https://github.com/bahmutov/find-webpack/blob/master/src/index.js - https://github.com/bahmutov/cypress-react-unit-test/blob/feature/cypress-mount-mode/plugins/cra-v3/file-preprocessor.js#L17

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            loaders: {
+              // you need to specify `i18n` loaders key with
+              // `vue-i18n-loader` (https://github.com/kazupon/vue-i18n-loader)
+              i18n: '@kazupon/vue-i18n-loader'
+            }
+          }
+        }
+      ]
+    }
+  }
```
Comment (bahmutov) [inline webpack.config.js:27]: can you set up your text editor to output a blank line at the end of the file, please?

File: preprocessor/webpack.js (hunk 1 above)
Comment (JessicaSachs) [reply preprocessor/webpack.js:12]: Can't because I need the `plugins` section, which cleanForCypress deletes

File: webpack.config.js (hunk 2 above)
Comment (JessicaSachs) [reply webpack.config.js:27]: Done.

File: preprocessor/webpack.js (hunk 1 above)
Comment (bahmutov) [reply preprocessor/webpack.js:12]: I have added a flag to clean method to mimic what you are doing and only remove problematic chunks plugins: https://github.com/bahmutov/find-webpack/blob/master/src/clean-for-cypress.js#L71

Maybe we can move your logic there and have a flag for Vue?

Comment (JessicaSachs) [reply preprocessor/webpack.js:12]: Will refactor again after things go green

**Pull request merged**